docs/extending.txt

```diff
@@ -549,3 +549,44 @@ arguments.  Example use:
         endswith=endswith) + t
 
     site.template_hooks['page_header'].append(greeting, True, u'Nikola Tesla', endswith=u'!')
+
+Shortcodes
+==========
+
+Some (hopefully all) markup compilers support shortcodes in these forms::
```

Comment:
Do we support it in HTML and ipynb?  (ipynb might need some extra help, I’d rather not run this over Python cells/a JSON file)